### PR TITLE
[RASDLG] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian for review

File: dll/win32/rasdlg/lang/ro-RO.rc [hunk 1]
```diff
@@ -2296,24 +2296,24 @@ BEGIN
     7321 "Introduceți un nume de autentificare pentru noul utilizator sau apăsați „Anulează”.\0"
     7322 "Parola introdusă este prea scurtă.\0"
     7323 "Parolele introduse nu corespund.  Reintroduceți-le.\0"
-    7324 "A survenit o eroare internă.  \0"
+    7324 "A apărut o eroare internă.  \0"
```
[the remark this answers is not in the document]
Comment (AndreiMiloiu) [reply dll/win32/rasdlg/lang/ro-RO.rc:2299]: I saw that there is something similar to the English file, too. That is the reason I didn't remove any extra space. Are there typos in the English files?

No, I don't know the answer. This is why I changed nothing regarding to the extra spaces.

[the remark this answers is not in the document]
Comment (AndreiMiloiu) [reply dll/win32/rasdlg/lang/ro-RO.rc:2299]: I understand. I will remove the extra spaces.

[the remark this answers is not in the document]
Comment (JoachimHenze) [reply dll/win32/rasdlg/lang/ro-RO.rc:2299]: No problem to wait with that one or two days, Maybe others can contribute or would have a reasonable explanation for those double-spaces. Up to now I raised that just as a question. If others think too that it is a bug, then we can and should fix all of them.

[the remark this answers is not in the document]
Comment (JoachimHenze) [reply dll/win32/rasdlg/lang/ro-RO.rc]: Ok, I will wait for other reviewers here. If they confirm that being a bug, then I will open a follow-up PR that will fix them in all other languages (aside ro-RO).

**Pull request merged**